book/website/community-handbook/bookdash/bookdash-events.md

```diff
@@ -0,0 +1,142 @@
+(ch-bookdash-events)=
+# The Book Dash Main Event Plans
+
+This subchapter provides information on the events that are organised for the book dash attendees.
+These events include pre-event drop-in calls, Day-1 brainstorming session and group dinner, and Day-2 working day, which are explained below.
+
+## Pre-Event drop-in calls
+
+The core team members hosts 2 pre-event drop-in calls, minimum one week before the main event to ensure that all the participants are made familiar with project, the format of the book dash, and how they can be contributing through their involvement at the book dash.
+It is an opportunity for the organising team and the selected attendees to get to know each other and discuss any concerns that they may have prior to the book dash.
+
+Several participants often come to these calls with some plans and topics they would like to explore for their contributions in _The Turing Way_.
+These topics can be something that is either missing in the existing chapters or need to be updated in the project.
+We also discuss _The Turing Way_ as a project in general and what the core team is working on to support the development of its guide on reproducibility, ethics, project design, collaboration, communication and a meta-book on the project itself.
+Most importantly, the core team goes through the contribution guideline and explain how attendees can effectively use their time at the book dash.
+
+The template for shared-notes can be accessed in the community handbook {ref}`here<ch-bookdash-precall-template>`.
+You can read notes from the calls from the most recent book dash [here](https://hackmd.io/@malvikasharan/bookdash).
+
+## Day-1: brainstorming and dinner
```

Comment:
Arguably a personal preference, but I would put the schedule overview first and then the expanded descriptions

Reply:
I will move the schedule above. It does make it easier to read that way.

Reply:
Moved the agenda to the top: https://github.com/alan-turing-institute/the-turing-way/pull/1262/commits/642fd193ae64a462e37ccd51a282c7112d703473